QuizWhois.Domain/Services/Implementations/QuestionService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using QuizWhois.Common.Models;
+using QuizWhois.Domain.Database;
+using QuizWhois.Domain.Entity;
+using QuizWhois.Domain.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuizWhois.Domain.Services.Implementations
+{
+    public class QuestionService : IQuestionService
+    {
+        public ApplicationContext _context { get; set; }
+
+        public QuestionService(ApplicationContext context)
+        {
+            _context = context;
+        }
+
+        public void AddQuestion(QuestionModel operationModel)
+        {
+            if (operationModel == null || operationModel.QuestionText == string.Empty || operationModel.CorrectAnswer == string.Empty)
+            {
+                throw new Exception("Operation Model was null");
```

Comment:
Throwing exceptions is a pricey solution for a simple validation. Should we discuss it?

Reply:
As long as model state is validating in controller, it is ok to throw exceptions here, as it would be really an exception